src/functionalTest/resources/sourceFiles/parent/judicial_appointments.csv

```diff
@@ -1,3 +1,3 @@
-per_Id,Base_Location_Id,Region_Id,Is_Principal_Appointment,Start_Date,End_Date,Active_Flag,Extracted_Date,Personal_Code,Appointment_Type_Id,Object_Id,Appointment
-1,747,,TRUE,1900-01-01 00:00:00,,TRUE,2019-12-04 11:29:43.480000000,111,1,,Magistrate
-2,1030,,TRUE,1900-01-01 00:00:00,,TRUE,2019-12-04 11:29:43.480000000,222,5,578256875287452,Circuit Judge
+per_Id,Base_Location_Id,Region_Id,Is_Principal_Appointment,Start_Date,End_Date,Active_Flag,Extracted_Date,Personal_Code,Appointment_Type_Id,Object_Id,Appointment,primary_location,secondary_location,tertiary_location,mrd_created_time,mrd_updated_time,mrd_deleted_time
+1,747,,TRUE,1900-01-01 00:00:00,,TRUE,2019-12-04 11:29:43.480000000,111,1,,Magistrate,,,,,,
```

Comment:
Can you please add some data and validate the same in the functional test?

Reply:
it sis done.